syntax = "proto3";

package compiler_server;

message CompileResponseArgs {
  uint32 id  = 1;
  uint32 pid = 2;
}
